spec/models/activerecord_obstacle_course_spec.rb

```diff
@@ -317,19 +319,17 @@
     unordered_items = [unordered_item_1, unordered_item_2, unordered_item_3]
 
     # ----------------------- Using Ruby -------------------------
-    items = Item.all
-
-    ordered_items = items.map do |item|
-      item if item.orders.present?
-    end.compact
-
-    ordered_items_names = ordered_items.map(&:name)
+    # items = Item.all
+    #
+    # ordered_items = items.map do |item|
+    #   item if item.orders.present?
+    # end.compact
+    #
+    # ordered_items_names = ordered_items.map(&:name)
     # ------------------------------------------------------------
-
     # ------------------ ActiveRecord Solution ----------------------
-    # Solution goes here
-    # When you find a solution, experiment with adjusting your method chaining
-    # Which ones are you able to switch around without relying on Ruby's Enumerable methods?
+    # ordered_items_names = Item.joins(:orders).uniq.pluck(:name)
+    ordered_items_names = Order.joins(:items).uniq.pluck(:name)
```

Comment:
@s-espinosa @carmer are these the only two ways that this would work? Also, is there a preference for distinct vs. uniq?